src/game/game.cpp

```diff
@@ -5920,6 +5920,11 @@ void Game::playerChangeOutfit(uint32_t playerId, Outfit_t outfit, uint8_t isMoun
 		return;
 	}
 
+	if (player->isSupportOutfit()) {
```

Comment:
For some reason when changing to lookType 75 it is not toggling the mount to false and it's causing a crash.